src/ds/d400/d400-auto-calibration.cpp

```diff
@@ -1552,8 +1552,8 @@ namespace librealsense
 
     std::shared_ptr<ds_advanced_mode_base> auto_calibrated::change_preset()
     {
-        preset old_preset_values;
-        rs2_rs400_visual_preset old_preset;
+        preset old_preset_values{};
```

Comment:
Can you remind me the effect of it?
Without it , it will also call the default Ctor no?

Reply:
I don't think that this is a real issue, but I preferred to add this and not mark the issue as "not a bug" as I was already changing other issues in this file.

Reading the documentation, this issue is very confusing and have cyclic definitions. As I understand, `preset` is a struct so it will be "aggregate initialized". Each of its inner structs will be also "aggregate initialized" and the inner most fields "value initialized". Since they are native types they all should be initialized to static_cast<T>( 0 ).